mentoring/public/themes/lms.css

```diff
@@ -0,0 +1,18 @@
+.themed-xblock.mentoring .checkmark-incorrect {
+    color: #c1373f;
```

Comment:
@bradenmacdonald there's an explicit requirement to keep current DOM/CSS as intact as possible to avoid potential migration problems in future. There's a bit more detailed explanation on the ticket.